provision/acc_provision/apic_provision.py

```diff
@@ -27,6 +27,12 @@
     requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
 except Exception:
     pass
+#Try importing Mapping for python 3.10, if it fails default to the older version
```

Comment:
at line 30 after # there should be a space as travis is failing. (Just checked the job log for the respective PR travis build)